test/util/schemaform-utils.jsx

```diff
@@ -15,6 +15,9 @@ export class DefinitionTester extends React.Component {
     const uiSchema = props.uiSchema;
     const data = props.data;
     let schema = updateRequiredFields(props.schema, uiSchema, data, state);
+    if (props.definitions) {
+      schema.definitions = props.definitions;
```

Comment:
Actually, if we're merging, is there no need to check whether props.definitions is not null?